front_end/src/components/Footer.js

```diff
@@ -4,7 +4,7 @@ import './Footer.css';
 const Footer = () => {
   return (
     <div className="Footer">
-      <code>Contributors: Apekshya| Conner| James| Matt| Mathew| Micheal| Samita| Stephanie </code>
+      <code>Contributors: Apekshya| Conner| James| Matt| Mathew| Michael| Samita| Stephanie </code>
```

Comment:
Should this say "Hudson" or "Michael"?![bitmoji](https://render.bitstrips.com/v2/cpanel/7e3bc453-64fb-4a34-bcaa-481724e5d3d9-55f9f02b-152f-475c-8597-45aa5e70485a-v1.png?transparent=1&palette=1&width=246)

Reply:
@Michael-Hudson